bootstrap/modules/ingress-controller/main.tf

```diff
@@ -0,0 +1,22 @@
+resource "helm_release" "ingress_controller" {
+  chart      = "ingress-controller"
```

Comment:
we'r always using the specific implementation name, not the generic one.

it's `ingress-nginx` in this case